editor/code_editor.cpp

```diff
@@ -1798,9 +1802,12 @@ CodeTextEditor::CodeTextEditor() {
 	status_bar->set_h_size_flags(SIZE_EXPAND_FILL);
 	status_bar->set_custom_minimum_size(Size2(0, 24 * EDSCALE)); // Adjust for the height of the warning icon.
 
+	idle_time = EDITOR_GET("text_editor/completion/idle_parse_delay");
+	idle_time_with_errors = EDITOR_GET("text_editor/completion/idle_parse_delay_in_error_state");
```

Comment:
Not needed, because `update_editor_settings()` is called automatically when the editor is created.